features/src/cuda/install.sh

```diff
@@ -168,12 +169,19 @@ if [ "${INSTALLNVJPEG:-false}" = true ]; then
 fi
 
 if [ "${INSTALLCUDNN:-false}" = true ]; then
-    LIBCUDNN_PKG="libcudnn${CUDNN_VERSION}"
-    if test -n "$(apt-cache search ${LIBCUDNN_PKG} 2>/dev/null)" \
-    && apt-cache policy ${LIBCUDNN_PKG} 2>/dev/null | grep -q "+${cuda_tag}"; then
-        PKGS+=("${LIBCUDNN_PKG}=*+${cuda_tag}");
+    CUDNNVERSION="${CUDNNVERSION:-9}";
```

Comment:
The default in `devcontainer-feature.json` is 8. Should this match?

Reply:
Yeah it should. I mistakenly thought your commit had bumped the default to 9.

Reply:
Okay. I will update this accordingly:
```suggestion
    CUDNNVERSION="${CUDNNVERSION:-8}";
```